### PR TITLE
Content updation

### PR DESCRIPTION
**Pull request by harisibrahimkv**

### CODE REVIEW

**Comment by harisibrahimkv** [comment]
@iambibhas @kracekumar Do take a look at this when you get time. Just some content updates and linting fixes.

**Comment by iambibhas** [comment]
Were the board member names there? Not sure removing them is a good idea. Also, should put the Travis build image url on the readme. Somebody pushed code to develop that was badly formatted.

**Comment by harisibrahimkv** [comment]
@iambibhas Interesting. That was not my doing. I think the merge that happened while doing the rebase screwed that up. Will fix it.

**Comment by iambibhas** [comment]
This is the commit https://github.com/harisibrahimkv/pssi.org.in/commit/d3f2adfb80cf501c5be7603b5bbf98d1c9a9c34d

**Comment by harisibrahimkv** [comment]
This is a commit as well: https://github.com/harisibrahimkv/pssi.org.in/commit/cecf9d03d500ed82588972c99d297bcc16115e5a

I had just put a full-stop at the end of Joint secretary. The rebase merge pulled in my content over the edited commit on develop, thus overwriting it. I am fixing it.

**Comment by harisibrahimkv** [comment]
@iambibhas Have fixed it.

File: templates/about.html [hunk 1]
```diff
@@ -27,13 +27,13 @@ <h3 class="push-1-top">PSSI and PSF (<a target="_blank" href="https://www.python
         <p>It is also worth mentioning that PSSI has received funds from PSF for conducting PyCon India in the past and might receive such funds in the future as well. However PSF has no say in the governance and conduct of the board members of PSSI.</p>
 
         <h3 class="push-1-top">PSSI Board</h3>
-        <p>The board positions of PSSI </p>
+        <p>The board positions of PSSI to which annual elections are held every year include,</p>
```
Comment (kracekumar) [inline templates/about.html]: As per new bye law election will be held every two years. I think for now can you omit this sentence ?

**Comment by harisibrahimkv** [comment]
@kracekumar Have fixed the annual election part.

**Pull request merged**